pytests/test_debugging.py

```diff
@@ -349,10 +349,10 @@ def testDeployConnectStartDisconnect(env: Env):
     env.assertEqual(len(toDictionary(env.cmd('TFUNCTION', 'list', 'library', 'lib', 'v'))), 0)
 
 
-@gearsTest(debugServerAddress="127.0.0.1:9005")
+@gearsTest(debugServerAddress="127.0.0.1:9006")
 def testSetBreakpointAndHitIt(env: Env):
     deploy_script(env)
-    client = DebuggerClient(env)
+    client = DebuggerClient(env, "ws://127.0.0.1:9006")
```

Comment:
I do not like this approach, we had this on redismodule-rs and we always forgot to avoid collisions. Can we think of a better way to distribute the ports?

Reply:
Fixed.